converter/src/test/java/gov/cms/qpp/acceptance/QualityMeasureIdRoundTripTest.java

```diff
@@ -76,4 +79,19 @@ public void testMeasureCMS160v5PerformanceRateUuid() throws IOException {
 		assertThat(details).comparingElementsUsing(DetailsMessageEquals.INSTANCE)
 				.contains(MipsQualityMeasureIdValidator.SINGLE_PERFORMANCE_RATE);
 	}
+
+	@Test
+	public void testMeasureCMS52v5WithInsensitiveTextUuid() throws IOException {
+		Converter converter = new Converter(new PathQrdaSource(INSENSITIVE_TEXT_FILE));
+		List<Detail> details = new ArrayList<>();
+
+		try {
+			converter.transform();
+		} catch (TransformException exception) {
+			AllErrors errors = exception.getDetails();
+			details.addAll(errors.getErrors().get(0).getDetails());
```

Comment:
I would prefer to make this test a bit more specific.  Can we use the special Google Truth thing we created to test validation errors to test that we do _not_ get a specific error that would occur if we accidentally became case sensitive again?

Reply:
I refactored this test to specifically assert for the missing `Electronic measure uuid` message.
I added another test to assert for measure data uuid's being incorrect as well.